dataGraphs/thematics/CreativeWork/graphs/local-contexts-notice.json

```
@@ -0,0 +1,55 @@
+{
+    "@type": "CreativeWork",
+    "identifier": "{Notice ID}",
+    "name": [
+        {
+            "@language": "{language_tag}",
+            "@value": "{Notice Title}"
+        },
+        {
+            "@language": "{language_tag}",
+            "@value": "{Notice Title}"
+        }
+    ],
+    "author": {
+        "@type": "Organization",
+        "identifier": "https://oceanexpert.org/institution/xxxxxxx",
+        "legalName": "Local Contexts Inc.",
+        "name": "Local Contexts",
+        "url": "https://localcontexts.org/"
+    },
+    "text": [
+        {
+            "@language": "{language_tag}",
+            "@value": "{Notice Text}"
+        },
+        {
+            "@language": "{language_tag}",
+            "@value": "{Notice Text}"
+        }
+    ],
+    "url": "{NOTICE_PAGE_URL}",
+    "image": [
+        {
+            "@type": "ImageObject",
+            "url": "{ICON_IMG}",
+            "encodingFormat": "image/png"
+        },
+        {
+            "@type": "ImageObject",
+            "url": "{ICON_SVG}",
+            "encodingFormat": "image/svg"
+        }
+    ],
+    "dateCreated" : "{date Label created}",
+    "sdDatePublished": {"@type": "https://schema.org/Date"},
+    "sdLicense": [
+        {"@type": "https://schema.org/CreativeWork"},
+        {"@type": "https://schema.org/URL"}
+    ],
+    "sdPublisher": [
+        {"@type": "https://schema.org/Organization"},
+        {"@type": "https://schema.org/Person"}
+    ],
+    "creditText": "text: recommended citation"
+}
```

For all Notices, license should be CC BY-NC-ND
```suggestion
    "license": "https://creativecommons.org/licenses/by-nc-nd/4.0/"
}
```